libs/utils/energy_model.py

```diff
@@ -1042,13 +1042,17 @@ def simplified_em_present_in_cpusysfs(target, directory=None):
         # em_loaders dictionary joins EM loaders and the identifying functions
         # with any associated metadata
         em_loaders = {
-            'sd'    : { 'check': em_present_in_sd,
-                        'load': cls.from_sd_target,
-                        'filename': '/proc/sys/kernel/sched_domain/cpu{}/domain{}/group{}/energy/{}' },
-            'sysfs' : { 'check': simplified_em_present_in_cpusysfs,
-                        'load': cls.from_simplifiedEM_target,
-                        'directory': '/sys/devices/system/cpu/energy_model' }
-                     }
+            'sched domain' : {
+                'check': em_present_in_sd,
+                'load': cls.from_sd_target,
+                'filename': '/proc/sys/kernel/sched_domain/cpu{}/domain{}/group{}/energy/{}'
+            },
+            'CPU' : {
```

Comment:
'CPU sysfs' ? `'Loading energy model from CPU data` doesn't great as-is IMO. Or ideally just log the directory name ...